### DISCUSSION TITLE
Multiple blank Claims fields spawned when adding an Advanced Claim to Group or Advanced Claim to Role? (possible bug)

### DISCUSSION THREAD

**Discussion by alansonsci** [keycloak/keycloak#16811] [Q&A]
Has anyone come across this issue when adding Identity Provider Mapper?
Multiple blank Claims fields get spawned when adding an Advanced Claim to Group or Advanced Claim to Role? (possible bug)

![image](https://user-images.githubusercontent.com/104139389/216456108-22e8b3b2-08f6-4cae-9d6f-c52f9998249b.png)

Demo of issue:
https://youtu.be/NTZWXBZ8WwI

**Comment by jonkoops** [comment]
I've created an issue out of this so it can be investigated (see https://github.com/keycloak/keycloak-ui/issues/4314).

**Reply by alansonsci** [reply]
I tested this with 20.0.3 and it seems like it's not a problem so we can close this.
